File: IntegrationExamples/ObjCAppWithCarthage/ObjCAppWithCarthage/ViewController.m
//
//  ViewController.m
//  ObjCAppWithCocoaPods
//
//  Copyright © 2022 Radar Labs, Inc. All rights reserved.
//

#import "ViewController.h"
@import RadarSDK;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    self.versionNumberLabel.text = Radar.sdkVersion;
}


@end

File: IntegrationExamples/SwiftAppWithCocoaPods/SwiftAppWithCocoaPods/ViewController.swift
//
//  ViewController.swift
//  SwiftAppWithCocoaPods
//
//  Copyright © 2022 Radar Labs, Inc. All rights reserved.
//

import RadarSDK
import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var versionNumberLabel: UILabel!

    override func viewDidLoad() {
        super.viewDidLoad()

        versionNumberLabel.text = Radar.sdkVersion
    }

}
